content/docs/start/experiments.md

```diff
@@ -170,14 +170,81 @@ $ git add dvc.lock params.yaml prc.json roc.json scores.json
 $ git commit -a -m "Preserve best random forest experiment"
 ```
 
-> `dvc push` only uploads persistent experiments that have been committed to
-> Git. The other experiments will not be pushed to the remote. See
-> `dvc exp push` and `dvc exp pull` for how to share other experiments.
+## Sharing experiments
+
+After committing the best experiment to our Git branch, we can
+[store and share](/doc/start/data-versioning#storing-and-sharing) it remotely
+like any other iteration of the pipeline.
+
+```dvc
+dvc push
+git push
+```
+
+<details>
+
+### 💡 Expand to try out sharing with simple remotes.
+
+A DVC remote stores the experiment data, and a Git remote stores the code,
+parameters, and other metadata associated with the experiment. DVC supports
+various types of remote storage (local file system, SSH, Amazon S3, Google Cloud
+Storage, HTTP, HDFS, etc.). The Git remote is often a central Git server
+(GitHub, GitLab, BitBucket, etc.). Let's set up both the DVC and Git remotes on
+our local filesystem to try them out:
+
+```dvc
+$ mkdir -p /tmp/dvcstore
+$ dvc remote add -d myremote /tmp/dvcstore
+$ git commit .dvc/config -m "Configure local remote"
+$ mkdir -p /tmp/gitremote
+$ git clone . /temp/gitremote
+$ git remote add gitremote /tmp/gitremote
+$ dvc push
+$ git push -u gitremote master
+```
+
+`dvc push` saves the promoted experiment data to the default DVC remote in
+`/tmp/dvstore`.
+
+`git push -u gitremote master` saves the code, parameters, and associated
+metadata from the experiment committed to our `master` branch to the Git remote
+named `gitremote` in `/tmp/gitremote`.
+
+Read on to learn how to share other experiments with DVC and Git remotes.
+
+</details>
+
+Experiments that have not been made a persistent part of the pipeline will not
+be stored or shared remotely through `dvc push` or `git push`.
```

Comment:
> persistent part of the pipeline

Each experiment contains a full pipeline version right? They're not parts of a DVC pipeline I think:

```suggestion
Experiments that have not been made persistent will not
be stored or shared remotely through `dvc push` or `git push`.
```

> Needs reformat